client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -38,6 +39,21 @@ class EditorConfirmationSidebar extends React.Component {
 		this.props.onPublish( true );
 	};
 
+	getBusyButtonLabel( publishButtonStatus ) {
+		switch ( publishButtonStatus ) {
+			case 'update':
+				return this.props.translate( 'Updating...' );
```

Comment:
Looks like the linter wants an ellipsis (`opt+semicolon` on Mac) instead of three periods.